docs/operate/control/viam-app.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Create a Viam app"
+linkTitle: "Create a Viam app"
+weight: 5
+layout: "docs"
+type: "docs"
+description: "Create and deploy custom web interfaces for your machines without managing hosting and authentication."
+---
+
+Create and deploy custom web interfaces for your machines without managing hosting and authentication.
+Once deployed, apps are accessible from a dedicated URL (`appname_publicnamespace.viamapplications.com`) and hosting and authentication is handled for you.
+
+When opening an app, users log in and then select a machine they have access to.
+Then your app is rendered and ready for use.
+
+{{<gif webm_src="/spa.webm" mp4_src="/spa.mp4" alt="Sample web app" max-width="500px">}}
+
+## Requirements
+
+{{< expand "Install the Viam CLI and authenticate." >}}
+Install the Viam CLI using the option below that matches your system architecture:
+
+{{< readfile "/static/include/how-to/install-cli.md" >}}
+
+Then authenticate your CLI session with Viam using one of the following options:
+
+{{< readfile "/static/include/how-to/auth-cli.md" >}}
+
+{{< /expand >}}
+
+## Build a custom web interface
+
+You can build a custom web interface to access your machines using your preferred framework like React, Vue, Angular, or others.
+
+### Access machines from your app
+
+Viam apps provide access to a machine by placing its API key in your cookies.
```

Comment:
```suggestion
Viam single-page apps provide access to a machine by placing its API key in your cookies.
```
Also, to whom do they provide access? Is this how the apps themselves access the machines? Consider rewriting for less ambiguity